website/docs/installation.mdx

```diff
@@ -0,0 +1,520 @@
+---
+title: Installation
+sidebar_position: 1
+hide_title: true
+---
+
+import TierLabel from "./_components/TierLabel";
+
+## Installing Weave GitOps
+
+This section details the steps required to install Weave GitOps on a Kubernetes cluster.
+
+### Pre-requisites
+
+#### Kubernetes Cluster
+Weave GitOps is compatible with conformant Kubernetes distributions which match the minimum required version level of [Flux](https://fluxcd.io/docs/installation/#prerequisites).
+
+#### Install Flux
+Weave GitOps is an extension to Flux and therefore requires that Flux has already been installed on your Kubernetes cluster. Full documentation is avilable at: [https://fluxcd.io/docs/installation/](https://fluxcd.io/docs/installation/).
+
+#### Configure access to the GitOps Dashboard web UI
+Weave GitOps includes a web UI which runs on your Kubernetes cluster. In order to allow users to access this, you must first configure an appropriate login mechanism. We support integration with OIDC providers, as well as an admin cluster user for getting started and emergencies. This cluster user can be disabled if preferred.
+
+Follow the guide here to appropriately configure access : [Securing access to the dashboard](./configuration/securing-access-to-the-dashboard).
+
+### Install the Helm Chart
+Weave GitOps is provided through a Helm Chart and installed as a Flux resource through a `HelmRepository` and `HelmRlease`. To install on your cluster, adjust the following where marked `<UPDATE>` based on the previous step, and commit the file to the location bootstrapped with Flux so that it is synchronized to your Cluster.
+
+```
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: ww-gitops
+  namespace: flux-system
+spec:
+  chart:
+    spec:
+      chart: weave-gitops
+      sourceRef:
+        kind: HelmRepository
+        name: ww-gitops
+  interval: 1m0s
+  values:
+    additionalArgs:
+    - --insecure
+    adminUser:
+      create: true
+      password: <UPDATE>
+      username: <UPDATE>
+    namespace: flux-system
+    rbac:
+      create: true
+---
+apiVersion: source.toolkit.fluxcd.io/v1beta1
+kind: HelmRepository
+metadata:
+  name: ww-gitops
+  namespace: flux-system
+spec:
+  interval: 1m0s
+  url: https://helm.gitops.weave.works
+```
+
+## Installing the gitops CLI
+
+The `gitops` command line interface provides a set of commands to make it easier to interact with Weave GitOps, including both the free open source project, and commercial Weave GitOps Enterprise product. It is currently supported on Mac (x86 and Arm), and Linux including WSL.
+
+Windows support is a [planned enhancement](https://github.com/weaveworks/weave-gitops/issues/663).
+
+To install the `gitops` CLI, please follow the following steps:
+
+```console
+curl --silent --location "https://github.com/weaveworks/weave-gitops/releases/download/v0.6.2/gitops-$(uname)-$(uname -m).tar.gz" | tar xz -C /tmp
```

Comment:
Is this version supposed to be hard-coded? :thinking:

Reply:
The release process says to update this manually after a successful release